larsim/IonizationScintillation/ISCalcCorrelated.h

```diff
@@ -48,14 +50,23 @@ namespace larg4 {
     double fRecombk;          ///< from LArG4Parameters service
     double fModBoxA;          ///< from LArG4Parameters service
     double fModBoxB;          ///< from LArG4Parameters service
+    double fLarqlChi0A;       ///< from LArG4Parameters service
+    double fLarqlChi0B;       ///< from LArG4Parameters service
+    double fLarqlChi0C;       ///< from LArG4Parameters service
+    double fLarqlChi0D;       ///< from LArG4Parameters service
+    double fLarqlAlpha;       ///< from LArG4Parameters service
+    double fLarqlBeta;        ///< from LArG4Parameters service
     bool fUseModBoxRecomb;    ///< from LArG4Parameters service
+    bool fUseModLarqlRecomb;  ///< from LArG4Parameters service
 
     const spacecharge::SpaceCharge* fSCE;
     const detinfo::LArProperties* fLArProp;
 
     void CalcIon(sim::SimEnergyDeposit const& edep);
     void CalcScint(sim::SimEnergyDeposit const& edep);
     double GetScintYieldRatio(sim::SimEnergyDeposit const& edep);
+    double chi0(double dEdx);
+    double fcorr(double EF, double dEdx);
```

Comment:
Would it be possible to use more descriptive function names and make the capitalisation consistent with the rest of the codebase?

Reply:
I guess the other functions also pass const arguments as well.  If you update the naming/capitalisation, you could consider passing the arguments as const values too.

Reply:
The issue of `const` correctness is a long discussion.  The general gist: in a function _declaration_, there is no benefit in marking variables passed in by value as `const`.  However, there _is_ a benefit to marking such variables as `const` in  the function _definition_.  So Dom's comment about `const` variables is partially correct.  A good pattern to follow:

```.cpp
int foo(int bar); //  Function declaration - 'bar' is not marked const
int foo(int const bar) { return bar + 1; } // Function definition - 'bar' is marked const
```